app/home/templates/includes/property_listing_weekly.html

```diff
@@ -39,6 +46,18 @@ <h5>{{property.name|title}}</h5>
                     
                         </div>
                     </div>
+                    <div class="container m-4">
+                      {% for page_num in properties.iter_pages(left_edge=1, right_edge=1, left_current=1, right_current=5) %}
+                        {% if page_num %}
+                            {% if properties.page == page_num %}
+                              <a class="btn btn-outline-info mb-4 disabled" href="{{ url_for('home_blueprint.index', page=page_num) }}">{{ page_num }}</a>
+                            {% else %}
+                              <a class="btn btn-info mb-4 " href="{{ url_for('home_blueprint.index', page=page_num) }}">{{ page_num }}</a>
+                            {% endif %}
+                        {% else %}
+                        {% endif %}
+                      {% endfor %}
+                    </div>
```

Comment:
same here you can replace this with this block

{% include "includes/pagination.html" %}

Reply:
Resolved